.github/workflows/pr-go.yaml

```diff
@@ -104,6 +104,8 @@ jobs:
         run: go install github.com/golang/protobuf/protoc-gen-go@${{ env.PROTOC_GEN_GO_VERSION }}
       - name: Generate proto go files
         run: make proto-go
+      - name: Generate proto go descriptor files
+        run: make proto-go-descriptor
```

Comment:
The `proto-go-description` rule creates the descriptor if needed, but it doesn't return an error if there is a diff.
We should do something similar to this.

https://github.com/bucketeer-io/bucketeer/blob/main/Makefile#L65-L67

Reply:
I just noticed that you are checking the diff by executing the `update-repos-check`.
It would be better to be more explicit by creating a rule to check it separately to keep consistency.

Reply:
Thanks, I fixed it 👍